query-translator/tests/translate.rs

```diff
@@ -72,8 +72,8 @@ fn test_rel() {
     let input = r#"[:find ?x :where [?x :foo/bar "yyy"]]"#;
     let parsed = parse_find_string(input).expect("parse failed");
     let algebrized = algebrize(&schema, parsed);
-    let select = cc_to_exists(algebrized.cc);
-    let SQLQuery { sql, args } = select.to_sql_query().unwrap();
-    assert_eq!(sql, "SELECT 1 FROM `datoms` AS `datoms00` WHERE `datoms00`.a = 99 AND `datoms00`.v = $v0");
+    let select = query_to_select(algebrized);
```

Comment:
Further to my earlier point, can I see `[:find ?x ?x ... ]` in these tests as well?